packages/data-store/src/configs.ts

```diff
@@ -1,74 +1,27 @@
-import {DataStoreConfigs, SDKConfigs} from './types/types';
-import {getDataSource} from './typeorm';
-import {SDKConfigsEntity} from './typeorm/entities/sdk-configs.entity';
-
-export async function getSDKConfigs(
-  options: DataStoreConfigs,
-): Promise<SDKConfigs> {
-  let result = await getDataSource().getRepository(SDKConfigsEntity).find();
-
-  if (result.length === 1) {
-    return result[0];
-  }
-
-  if (result.length > 1) {
-    throw new Error('More than one SDKConfigsEntity found');
-  }
-
-  const sdkConfigs = {
-    id: 'sdk-configs',
-    activeWallet: options.sdkConfigs.activeWallet,
-    version: null,
-  };
-
-  await getDataSource().getRepository(SDKConfigsEntity).save(sdkConfigs);
-
-  return sdkConfigs;
-}
-
-export const defaultDataStoreConfigs: DataStoreConfigs = {
-  sdkConfigs: {
-    activeWallet: 'mainnet',
-  },
+import {DataStoreConfigs} from './types';
+import {genericDocumentNetworkResolver} from './document-network-resolver';
+
+export const DEFAULT_CONFIGS: DataStoreConfigs = {
+  defaultNetwork: 'mainnet',
+  nonIsolatedNetworks: false,
+  databasePath: 'dock-wallet-sdk.db',
+  documentNetworkResolver: genericDocumentNetworkResolver,
   networks: [
     {
       name: 'Mainnet',
       id: 'mainnet',
+      configs: {
```

Comment:
how would we inject new networks?

Reply:
I've created an example here https://github.com/docknetwork/react-native-sdk/pull/115/commits/3e99e343b144e0eb17d7e9db1dd43fba748d184b